File: 2024-2/sesiones-de-clase/sesion-de-clase-23-2/src/Alumno.java
/**
 *
 * @author erichuiza
 */
class Alumno {
    private int codigo;
    private String nombre;
    private int ciclo;
    
    public Alumno() {
    }
    
    public Alumno(int codigo, String nombre, int ciclo) {
    }
    
    public int getCodigo() {
        return codigo;
    }

    public String getNombre() {
        return nombre;
    }

    public int getCiclo() {
        return ciclo;
    }
    
    public void setCodigo(final int codigo) {
        this.codigo = codigo;
    }

    public void setNombre(final String nombre) {
        this.nombre = nombre;
    }

    public void setCiclo(final int ciclo) {
        this.ciclo = ciclo;
    }
    
    
}

File: 2026-1/capitulo-03/stl/ejercicio02/Alumno.cpp
//
// Created by Eric Huiza on 6/12/26.
//

#include "Alumno.h"

int Alumno::getCodigo() const {
    return codigo;
}

string Alumno::getNombre() const {
    return nombre;
}

int Alumno::getEdad() const {
    return edad;
}

int Alumno::getCiclo() const {
    return ciclo;
}

string Alumno::getFacultad() const {
    return facultad;
}

void Alumno::setCodigo(int codigo) {
    this->codigo = codigo;
}

void Alumno::setNombre(string nombre) {
    this->nombre = nombre;
}

void Alumno::setEdad(int edad) {
    this->edad = edad;
}
void Alumno::setCiclo(int ciclo) {
    this->ciclo = ciclo;
}

void Alumno::setFacultad(string facultad) {
    this->facultad = facultad;
}

vector<Curso> Alumno::getCursos() const {
    return cursos;
}

void Alumno::setCursos(vector<Curso> cursos) {
    this->cursos = cursos;
}